lessons/githubPages/lesson.md

```diff
@@ -1,37 +1,40 @@
 ---
 layout: page
 title: 'Web design with Jekyll/Github Pages'
```

Comment:
A small change: it's GitHub (there are several other capitalizations used below.)